pkg/operator/staticpod/controller/staticpodstate/staticpodstate_controller.go

```diff
@@ -57,7 +57,9 @@ func NewStaticPodStateController(
 	}
 	return factory.New().WithInformers(
 		operatorClient.Informer(),
+		// informers are needed here because their Getter are cached lister based
 		kubeInformersForTargetNamespace.Core().V1().Pods().Informer(),
+		kubeInformersForTargetNamespace.Core().V1().ConfigMaps().Informer(),
```

Comment:
is `configMapGetter` even used by this controller?

Reply:
it isn't. removing

Reply:
since the `configMapGetter` was used and you removed it why we need to sync `kubeInformersForTargetNamespace.Core().V1().ConfigMaps().Informer()` ?

Reply:
removed it again